README.rst

```diff
@@ -178,6 +153,69 @@ is ``helloworld`` in our case. Indeed:
    Hello John Doe!
    Hello Jane Doe!
 
+Local testing with CWL
+=========================
+
+Let us test whether the CWL workflow execution works locally as well.
+
+To prepare the execution, we can:
+
+- either place input files ``code/helloworld.py`` and ``inputs/names.txt`` into the directory with ``helloworld-job.yml``
```

Comment:
Instead of offering choice to users in the docs (note that users might not be very familiar with CWL), let us simplify the "Local testing with CWL" by standardising on one technique only.  I propose to emulated what we show for Yadage.  That is, create a new local directory called `cwl-local-run` (analogous to `yadage-local-run`) and copy any necessary inputs there and use this directory as `cwd` for the local CWL run.  This has an advantage that is does not pollute the usual repository structure when people try to run this.